src/components/CartContextWrapper/CartContextWrapper.js

```diff
@@ -1,98 +1,99 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 
 import { CartContext } from '../../util/context';
 
 export const CartContextWrapper = ({ children }) => {
-  const [cart, setCart] = useState([
-    {
-      id: 1,
-      title: 'Macbook pro 14 inches',
-      imageURL:
-        'https://store.storeimages.cdn-apple.com/4982/as-images.apple.com/is/mbp14-spacegray-select-202110?wid=904&hei=840&fmt=jpeg&qlt=80&.v=1632788573000',
-      description: 'lorem ipsam',
-      unitPrice: 1999.99,
-      quantity: 1,
-    },
-  ]);
+    const [cart, setCart] = useState([]);
 
-  const calculateTotalItemsInCart = () => {
-    let totalItem = 0;
-    try {
-      cart.forEach((product) => {
-        totalItem = totalItem + parseInt(product.quantity);
-      });
-      return totalItem;
-    } catch (error) {
-      console.log(error);
-    }
-  };
+    useEffect(() => {
+        if (window.localStorage.getItem('cart')) {
+            setCart(JSON.parse(window.localStorage.getItem('cart')));
+        }
+    }, []);
 
-  const calculateTotalPriceInCart = () => {
-    let totalPrice = 0;
-    try {
-      cart.forEach((product) => {
-        totalPrice =
-          totalPrice + product.quantity * parseFloat(product.unitPrice);
-      });
-      return totalPrice.toFixed(2);
-    } catch (error) {
-      console.log(error);
-    }
-  };
+    const calculateTotalItemsInCart = () => {
+        let totalItem = 0;
+        try {
+            cart.forEach((product) => {
+                totalItem = totalItem + parseInt(product.quantity);
+            });
+            return totalItem;
```

Comment:
if you want you can simply use reduce method like this

`return cart.reduce((accum, product) => accum + parseInt(product.quantity), 0)`

Reply:
done